Formula/imagemagick.rb

```diff
@@ -72,49 +72,149 @@ def install
       --enable-static
     ]
 
+    if build.without? "freetype"
```

Comment:
I was using `--without` for recommended options and `--with` otherwise. No problem with always using `--with`.